wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/devfile/convert/DevfileConverter.java

```diff
@@ -114,7 +115,8 @@ public DevfileImpl workspaceToDevFile(WorkspaceConfigImpl wsConfig)
 
   public WorkspaceConfig convert(Devfile devfile) throws ServerException {
     try {
-      return devFileToWorkspaceConfig(new DevfileImpl(devfile), urlFileContentProvider);
+      return devFileToWorkspaceConfig(
+          new DevfileImpl(devfile), new CachingProvider(urlFileContentProvider));
```

Comment:
There's a factory method `FileContentProvider.cached()` that's maybe slightly more readable

Reply:
Ok